src/3p-frame-messaging.js

```diff
@@ -14,10 +14,11 @@
  * limitations under the License.
  */
 
+import {devAssert} from './core/assert';
 import {internalListenImplementation} from './core/dom/event-helper-listen';
+import {rethrowAsync} from './core/error';
```

Comment:
nit: Update all of these to use `#core` instead of `./core`. It'll make moving the file a no-diff operation later. Really all files using this notation should be updated, but it hasn't been prioritized yet

Reply:
Good idea, I was wondering about that when my auto import for `rethrowAsync` put it in `#core` but the others were in `./core`. Updated :)